### ISSUE TITLE
companion 3.0

### ISSUE THREAD

**Issue by kgspurs** [bitfocus/companion-module-sennheiser-evolutionwireless#3]
hello, this module doesn't exist in the latest version of company, is there plans to add it?

**Comment by Dahlsound** [comment]
Yes! Would really love for this to be available in the Companion connections again. Is there otherwise a way to install it manually?
